actions/rules/action_nil.go

```diff
@@ -0,0 +1,17 @@
+package rules
+
+import (
+	"github.com/fabric8-services/fabric8-wit/actions/change"
+)
+
+// ActionNil is a dummy action rule that does nothing and has no sideffects.
+type ActionNil struct {
+}
+
+// make sure the rule is implementing the interface.
+var _ Action = ActionNil{}
```

Comment:
Implicitly satisfied interface is feature of Go, not a limitation.

Reply:
Sure. And a discussable one. But in this case, this makes sure the rule actually implements the interface as this is required for registering and using the role. If an issue would change the signature, this might result in a crash that is only appearing on runtime.

Reply:
> Implicitly satisfied interface is a feature of Go, not a limitation.

@baijum we have those compile checks for a very long time with the `convert.Equaler` interface as well. You can read more about this here: https://golang.org/doc/effective_go.html#blank_implements